components/embl-notifications/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### 1.2.1
+
+* Support added for GA region. his region is generic for notifications.
```

Comment:
fix "his" spelling typo

Reply:
"This region is generic for banner notifications"

Reply:
ok, fixed this typo